openhands/resolver/README.md

```diff
@@ -0,0 +1,182 @@
+     - `LLM_MODEL`: LLM model to use (e.g., "anthropic/claude-3-5-sonnet-20241022")
+     - `LLM_API_KEY`: Your LLM API key
+   - Optional:
+     - `LLM_BASE_URL`: Base URL for LLM API (only if using a proxy)
+
+   Note: You can set these secrets at the organization level to use across multiple repositories.
+
+6. Usage:
+   There are two ways to trigger the OpenHands agent:
+
+   a. Using the 'fix-me' label:
+      - Add the 'fix-me' label to any issue you want the AI to resolve
+      - The agent will consider all comments in the issue thread when resolving
+      - The workflow will:
+        1. Attempt to resolve the issue using OpenHands
+        2. Create a draft PR if successful, or push a branch if unsuccessful
+        3. Comment on the issue with the results
+        4. Remove the 'fix-me' label once processed
+
+   b. Using `@openhands-agent` mention:
+      - Create a new comment containing `@openhands-agent` in any issue
+      - The agent will only consider the comment where it's mentioned
+      - The workflow will:
+        1. Attempt to resolve the issue based on the specific comment
+        2. Create a draft PR if successful, or push a branch if unsuccessful
+        3. Comment on the issue with the results
+
+Need help? Feel free to [open an issue](https://github.com/all-hands-ai/openhands-resolver/issues) or email us at [contact@all-hands.dev](mailto:contact@all-hands.dev).
+
+## Manual Installation
+
+If you prefer to run the resolver programmatically instead of using GitHub Actions, follow these steps:
+
+1. Install the package:
+
+```bash
+pip install openhands-resolver
+```
+
+2. Create a GitHub access token:
+   - Visit [GitHub's token settings](https://github.com/settings/personal-access-tokens/new)
+   - Create a fine-grained token with these scopes:
+     - "Content"
+     - "Pull requests"
+     - "Issues"
+     - "Workflows"
+   - If you don't have push access to the target repo, you can fork it first
+
+3. Set up environment variables:
+
+```bash
+# GitHub credentials
+export GITHUB_TOKEN="your-github-token"
+export GITHUB_USERNAME="your-github-username"  # Optional, defaults to token owner
+
+# LLM configuration
+export LLM_MODEL="anthropic/claude-3-5-sonnet-20241022"  # Recommended
+export LLM_API_KEY="your-llm-api-key"
+export LLM_BASE_URL="your-api-url"  # Optional, for API proxies
+```
+
+Note: OpenHands works best with powerful models like Anthropic's Claude or OpenAI's GPT-4. While other models are supported, they may not perform as well for complex issue resolution.
+
+## Resolving Issues
+
+The resolver can automatically attempt to fix a single issue in your repository using the following command:
+
+```bash
+python -m openhands_resolver.resolve_issue --repo [OWNER]/[REPO] --issue-number [NUMBER]
+```
+
+For instance, if you want to resolve issue #100 in this repo, you would run:
+
+```bash
+python -m openhands_resolver.resolve_issue --repo all-hands-ai/openhands-resolver --issue-number 100
```

Comment:
There multiple instances of `openhands_resolver.resolve_issue`. I think they should all be updated to `openhands.resolver.resolve_issue` in the README.